1-exercises/A-array-find/exercise.js

```diff
@@ -9,15 +9,16 @@ let names = [
   "Rakesh",
   "Antonio",
   "Alexandra",
-  "Andronicus",
+  "Andronicu",
   "Annam",
   "Mikey",
   "Anastasia",
   "Karim",
   "Ahmed",
 ];
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+let longNameThatStartsWithA = names.find(
+  (element) => element.length > 7);
```

Comment:
this is almost right, good check for the name being longer than 7, but there should also be a check for the name beginning with A :)

Reply:
yeah , You are right. I have amended the code. thanks.